tests/v1/tpu/test_mha_attn.py

```diff
@@ -0,0 +1,109 @@
+# SPDX-License-Identifier: Apache-2.0
+"""
+Test:
+
+* Tests for MultiHeadAttention layer
+"""
+
+import pytest
+import torch
+import torch_xla
+import torch_xla.core
+import torch_xla.core.xla_model
+
+from vllm import envs
+from vllm.attention.layer import MultiHeadAttention
+from vllm.attention.selector import _cached_get_attn_backend
+from vllm.platforms import current_platform
+
+if not envs.VLLM_USE_V1:
```

Comment:
Shouldn't this also check for the default behavior env bar as well?

Reply:
I remember I discussed it with @robertgshaw2-redhat 🤔 
Maybe I can check if the env was set rather than its value?